dashboard/app/models/script.rb

```diff
@@ -1111,6 +1112,18 @@ def summarize(include_stages = true, user = nil)
     summary
   end
 
+  # @return {Hash<string,number[]>|number[]}
```

Comment:
It looks like the last bit of this type signature is wrong; just `{Hash<string,number[]>}`?

Reply:
Good catch! copy-pasta.